packages/amplify_authenticator/lib/src/l10n/country_resolver.dart

```diff
@@ -19,36 +19,373 @@ import 'package:flutter/material.dart';
 import 'authenticator_localizations.dart';
 import 'resolver.dart';
 
-enum CountryResolverKey { selectDialCode, unitedStates, unitedKingdom }
+enum CountryResolverKey {
+  selectDialCode,
+  noDialCodeSearchResults,
+  unitedStates,
+  unitedKingdom,
+  algeria,
+  andorra,
+  angola,
+  anquilla,
+  antiguaAndBarbuda,
+  argentina,
+  armenia,
+  aruba,
+  australia,
+  austria,
+  azerbaijan,
+  bahamas,
+  bahrain,
+  bangladesh,
+  barbados,
+  belarus,
+  belgium,
+  belize,
+  benin,
+  bermuda,
+  bhutan,
+  bolivia,
+  bosniaHerzegovina,
+  botswana,
+  brazil,
+  brunei,
+  bulgaria,
+  burkinaFaso,
+  burundi,
+  cambodia,
+  cameroon,
+  canada,
+  capeVerdeIslands,
+  caymanIslands,
+  centralAfricanRepublic,
+  chile,
+  china,
+  colombia,
+  comoros,
+  congo,
+  cookIslands
+}
 
 /// The resolver class for country data
 class CountryResolver extends Resolver<CountryResolverKey> {
   const CountryResolver();
 
-  /// The label for the United States.
   String selectDialCode(BuildContext context) {
     return AuthenticatorLocalizations.countriesOf(context).selectDialCode;
   }
 
-  /// The label for the United States.
+  String noDialCodeSearchResults(BuildContext context) {
```

Comment:
It would be nice if all these classes, fields, methods and getters can be automatically generated based on a `json` file or something similar.

Reply:
@HuiSF Made some progress on this, but I don't know when it will be production ready.

Reply:
@HuiSF We now (somewhat clunky) code that generates all the necessart country information.